pkg/version/version.go

```diff
@@ -191,3 +197,81 @@ func IsReleaseVersion() bool {
 		GetMainVersion() != "" &&
 		!strings.Contains(GetMainVersion(), "-")
 }
+
+// IsPriorToScannerV4 returns true if version represents a version of ACS from prior to the
+// introduction of Scanner V4. Will return an error if cannot determine result.
+func IsPriorToScannerV4(version string) (bool, error) {
+	var err error
+	parsed, err := parseVersion(version)
+	if err != nil {
+		return false, err
+	}
+
+	x := parsed.MarketingMajor
+	y := parsed.EngRelease
+	z := parsed.PatchLevel
+
+	if x < 4 || y < 3 || (y == 3 && (z == "" || z != "x")) {
```

Comment:
This fails for 5.0.0

Reply:
Indeed! Nice catch!

Reply:
https://github.com/stackrox/stackrox/pull/9803/commits/30c2528cf548279c94bc5378ba3d06de70e6edf8